lib/ansible/modules/monitoring/zabbix/zabbix_screen.py

```diff
@@ -414,8 +440,14 @@ def main():
             graphs_in_row = zabbix_screen['graphs_in_row']
             graph_width = zabbix_screen['graph_width']
             graph_height = zabbix_screen['graph_height']
-            host_group_id = screen.get_host_group_id(host_group)
-            hosts = screen.get_host_ids_by_group_id(host_group_id, sort)
```

Comment:
@D3DeFi Replaced old functions with the new ones, since naming matching better to what it does. Added similar sort function/feature to the new version of the function.

This should now be fixed.